web/src/components/Footer.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  Box,
+  Flex,
+  IconButton,
+  Text,
+  useColorMode,
+  useColorModeValue,
+} from "@chakra-ui/react";
+import React from "react";
+import { BsLinkedin } from "react-icons/bs";
+import { VscGithub, VscGithubInverted, VscTwitter } from "react-icons/vsc";
+
+import { MarkdownText } from "./MarkdownText";
+
+export const Footer = () => {
+  const { colorMode } = useColorMode();
+  const handleRedirects = (
+    e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+  ) => {
+    window.open(e.currentTarget.value, "_blank");
+  };
+
+  let gitHubIconButton = <VscGithub size="1.2em" />;
+
+  if (colorMode === "dark") {
+    gitHubIconButton = <VscGithubInverted size="1.2em" />;
+  }
+
+  return (
+    <>
+      <Box
+        bg={useColorModeValue("#ECE8FD", "#2F206E")}
+        justifyContent="center"
+        alignItems="center"
+        zIndex={5}
+        fontSize="0.85em"
+        padding="10px"
+      >
+        <Flex
+          direction="column"
+          justifyContent="center"
+          gap={0}
+          alignItems="center"
+        >
+          <MarkdownText>
+            RealTime Digital Marketing is a demo application running on
+            [**SingleStoreDB**](https://singlestore.com).
+          </MarkdownText>
```

Comment:
Can you use an anchor `<a>` for the link? We don't usually use Markdown in the Product, so maybe it's better to remove it from here, what do you think?
The style is different from the mockups, so it will probably be easier to fix it using the anchor

Reply:
I agree with Marta here. Avoid using `MardownText` unless it really simplifies things.  
You can take inspiration from our `Link` component in helios, although we probably don't need something as complex in here :raised_hands:

Reply:
That was pre implemented before 
Let me change that